### ISSUE TITLE
Object.entries

### ISSUE THREAD

**Issue by plaanupam** [airbnb/react-dates#405]
I'm getting this error on browsers which does not support Object.entries
```
TypeError: Object.entries is not a function. (In 'Object.entries(opts)', 'Object.entries' is undefined)
```
I looked into previous issues and found that the plugin is supported in Internet explorer also but Object.entries is not supported by IE.

Do I have to use polyfill separately to support Object.entries?

**Comment by smendozaisf** [comment]
I'm getting the same error running my tests. And it seems to be with airbnb react dates. Any solutions yet?

**Comment by ljharb** [comment]
Yes, it's  assumed that you are polyfilling with the equivalent of https://npmjs.com/airbnb-browser-shims

You can use https://npmjs.com/object.entries if that's the only one you need.

**Comment by ljharb** [comment]
However, I think it's probably a good idea to update airbnb-prop-types for this specific case; I'll have one released shortly.

**Comment by plaanupam** [comment]
I installed core-js and imported core-js/fn/object/entries module
```
import 'core-js/fn/object/entries';
```

It's size is around 1KB

**Comment by ljharb** [comment]
The size of polyfills is negligible, and everyone should be polyfilling everything; but enough people are getting caught with this issue that I've fixed it at its source. (see https://github.com/airbnb/prop-types/issues/10 and https://github.com/airbnb/prop-types/issues/9)

I've released `airbnb-prop-types` `v2.5.2` - an `npm install` should clear this up.
